site/en/guides/border-router/esp-br.md

```diff
@@ -0,0 +1,125 @@
+# ESP Thread Border Router
+
+Contributor: https://github.com/gjc13
+
+ESP Thread border router is a FreeRTOS based implementation running on a combination of Espressif’s Wi-Fi and 802.15.4 SoCs.
+
+Hardware requirements:
+
+* An ESP32 series Wi-Fi dev kit. (ESP32, ESP32-C3, ESP32-S2 or ESP32-S3)
+* An [ESP32-H2](https://www.espressif.com/en/news/ESP32_H2) dev kit for running RCP.
+
+## Hardware connection
+
+The ESP32-H2 RCP will be connected to the border router via UART.
+
+<figure>
+<a href="../../guides/images/otbr-esp-connection.jpg"><img src="../../guides/images/otbr-esp-connection.jpg" width="600" border="0" alt="ESP32-H2 UART" /></a><figcaption>ESP32-H2 UART connection</figcaption>
+</figure>
+<br/>
+
+* Connect the ground PIN of ESP32 and ESP32-H2
+* Connect PIN 4 of ESP32 to PIN TX of ESP32-H2
+* Connect PIN 5 of ESP32 to PIN RX of ESP32-H2
+
+## Setup ESP-IDF
+
+The code for running ESP Thread border router is shipped in the Espressif development framework, ESP-IDF.
+
+Please follow the [official installation guide](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/get-started/index.html#installation-step-by-step) to setup the environment.
+
+## Build and flash the RCP
+
+```
+cd ${IDF_PATH}/examples/openthread/ot_rcp
+idf.py --preview set-target esp32h2
+idf.py build flash
+```
+
+## Build and flash the border router
+
+### Config the Thread and the Wi-Fi network
+
+The border router will automatically join the Wi-Fi network and create a new Thread network if there is not one in its storage.
+The network parameters can be configured in the config menu:
+
+```
+cd ${IDF_PATH}/examples/openthread/ot_br
+idf.py menuconfig
+```
+
+The Wi-Fi network parameters can be configured in `Example Connection Configuration > connect using Wi-Fi interface`
+The Thread network parameters can be configured in the `OpenThread network parameters` menu.
```

Comment:
Can you make these two items a list?

Also, instead of putting the menu options in code font, can you use bold text.